pytorch_lightning/core/lightning.py

```diff
@@ -1316,7 +1171,7 @@ def optimizer_step(
         By default, Lightning calls ``step()`` and ``zero_grad()`` as shown in the example
         once per optimizer.
 
-        .. tip:: Consider using `manual_optimizer_step` instead of overriding this method as done previously.
+        .. tip:: Consider using `.step()` instead of overriding this method as done previously.
```

Comment:
```suggestion
        .. tip:: With `Trainer(enable_pl_optimizer=True)`, you can user `optimizer.step()` directly and it will handle zero_grad, accumulated gradients, AMP, TPU and more automatically for you.
```